### DISCUSSION TITLE
CSS Pseudo class

### DISCUSSION THREAD

**Discussion by FredH2O** [microsoft/Web-Dev-For-Beginners#1102] [General]
Hello Everyone,

I am a beginner and right now I am struggling with remember pseudo classes in CSS, is there any tips you can give for me to remember them better ? And are they that important to spend more time on them ? Thanks alot ! 🙏🙏

**Comment by Suryam1** [comment]
Hello You only need to remember some most often used pseudo classes rest you just have to go through so when you want to use them you wont face any issues with it. I hope it will help.

**Comment by abhijitkumarIN** [comment]
ya Mr @Dojodyo  there 're few most used pseudo class which 'll used mostly just try to understand only one properties with given link 
one basis of one example easily you 'll be able to get all pseudo class 

https://www.w3schools.com/css/css_pseudo_classes.asp   ---> basic  -- you must  preffered this one at first time  
https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes ---> advanced
